cmd/gosky/main.go

```diff
@@ -204,8 +208,14 @@ var readRepoStreamCmd = &cli.Command{
 			return h, nil
 		}
 
+		rr := cctx.Duration("read-delay")
+
 		rsc := &events.RepoStreamCallbacks{
 			RepoCommit: func(evt *comatproto.SyncSubscribeRepos_Commit) error {
+				if rr != 0 {
+					time.Sleep(rr)
+				}
+
```

Comment:
Could we use a `rate.limiter` here instead and then allow a user to configure a maximum throughput? That might be a bit easier to use.